nodes/nodes/eco-self-node-remediation-operator.adoc

```diff
@@ -23,6 +23,10 @@ include::modules/eco-configuring-machine-health-check-with-self-node-remediation
 
 include::modules/eco-self-node-remediation-operator-troubleshooting.adoc[leveloffset=+1]
 
+[id="about-collecting-data-snr"]
+== About collecting data for the Self Node Remediation Operator
```

Comment:
Suggestion:
Collecting Self Node Remediation operator data for failure debugging

Reply:
Thanks @anna-savina 
I am not sure if we need to be so specific. Debugging would include any debugging- including failures

Reply:
I mean that must-gather collect data for debugging or any other purpose, just not for Self Node Remediation Operator.
Maybe we can say:
Collecting data about the Self Node Remediation Operator

And the same for NMO and NHC